language/de/acp/common.php

```diff
@@ -373,9 +373,11 @@
 	'ERROR_MBSTRING_ENCODING_TRANSLATION'			=> 'Die transparente Zeichenkodierung ist fehlerhaft konfiguriert',
 	'ERROR_MBSTRING_ENCODING_TRANSLATION_EXPLAIN'	=> '<var>mbstring.encoding_translation</var> muss 0 sein. Du kannst den aktuellen Wert auf der Seite <samp>PHP-Information</samp> prüfen.',
 	'ERROR_MBSTRING_HTTP_INPUT'						=> 'Die HTTP-Eingabe-Kodierung ist fehlerhaft konfiguriert',
-	'ERROR_MBSTRING_HTTP_INPUT_EXPLAIN'				=> '<var>mbstring.http_input</var> muss auf <samp>pass</samp> eingestellt sein. Du kannst den aktuellen Wert auf der Seite <samp>PHP-Information</samp> prüfen.',
+	'ERROR_MBSTRING_HTTP_INPUT_EXPLAIN'				=> '<var>mbstring.http_input</var> muss leer gelassen sein. Du kannst den aktuellen Wert auf der Seite <samp>PHP-Information</samp> prüfen.',
```

Comment:
Vorschlag 1.:

> mbstring.http_input muss leer sein

oder 

> mbstring.http_input muss leer gelassen werden

Reply:
Ich bin da auch schon hängen geblieben. Es sollte halt zu dem Rest passen (muss auf ... eingestellt sein)
In der PHP-Dokumentation heißt es "PHP 5.6 and later users should leave this empty and set default_charset instead." (auch auf der deutschen Seite), womit die Formulierung "leer lassen" schon passt.

Ich wäre für "darf nicht gesetzt sein" oder "Für x darf kein Wert festgelegt sein"

Reply:
Dann:  "darf nicht gesetzt sein"